test/test-PageScraper.js

```diff
@@ -109,14 +109,33 @@ exports.test_blacklist = function(assert) {
   assert.equal(isAccepted, false, `${url} exists in the blacklist`);
 };
 
+exports.test_compute_image_sizes = function*(assert) {
+  const imageWidth = 350;
+  const imageHeight = 150;
+  let metadataObj = {
+    url: "https://www.hasAnImage.com",
+    images: [{url: `http://placehold.it/${imageWidth}x${imageHeight}`}]
```

Comment:
Is this expecting to do a network request? Probably best to use an data uri, e.g., 1x1 pixel:

`data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEAAAAALAAAAAABAAEAAAI=`